lib/sedna/README.md

```diff
@@ -1,2 +1,289 @@
 
-### Sedna Python SDK
+# Sedna Python SDK
```

Comment:
can this lib doc file be added in #159 instead?

Reply:
check